### PR TITLE
Add infinite earring

### PR DESCRIPTION
**Pull request by StevenClontz**
Addresses #792.

### CODE REVIEW

File: spaces/S000201/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- Hawaiian earring
+refs:
+- wikipedia: Hawaiian_earring
+  name: Hawaiian earring on Wikipedia
+- zb: "0951.54001"
+  name: Topology (Munkres)
+---
+
+The subspace of {S176} defined by
+$$X=\bigcup_{n=1}^{\infty}\left\{(x,y)\in\mathbb{R}^2\mid\left(x-\frac{1}{n}\right)^2+y^2=\left(\frac{1}{n}\right)^2\right\}.$$
```
Comment (StevenClontz) [inline spaces/S000201/README.md:14]:
```suggestion
$$X=\bigcup_{n=1}^{\infty}\left\{(x,y)\in\mathbb{R}^2\middle\mid\left(x-\frac{1}{n}\right)^2+y^2=\left(\frac{1}{n}\right)^2\right\}.$$
```

File: spaces/S000201/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Hawaiian earring
+refs:
+- wikipedia: Hawaiian_earring
+  name: Hawaiian earring on Wikipedia
+- zb: "0951.54001"
+  name: Topology (Munkres)
+---
+
+The subspace of {S176} defined by
+$$X=\bigcup_{n=1}^{\infty}\left\{(x,y)\in\mathbb{R}^2\middle\mid\left(x-\frac{1}{n}\right)^2+y^2=\left(\frac{1}{n}\right)^2\right\}.$$
```
Comment (StevenClontz) [inline spaces/S000201/README.md:14]:
```suggestion
$$X=\bigcup_{n=1}^{\infty}\left\{(x,y)\in\mathbb{R}^2:\left(x-\frac{1}{n}\right)^2+y^2=\left(\frac{1}{n}\right)^2\right\}.$$
```

**Comment by prabau** [comment]
Are you satisfied with the look on the web page (gigantic equation all by itself)?  Also, we could be slightly more descriptive with something like:
"The subspace of ... defined by $X=...$, that is, the union of the circles centered at $(\frac 1n,0)$ of radius $\frac 1n$ for $n=1,2,\dots$."

Approving and leaving it up to you if you want to make some changes.

**Review by prabau — APPROVED** [review]

**Comment by StevenClontz** [comment]
> we could be slightly more descriptive with something like:

I like this suggestion, thanks. And thanks for the proactive approval. Editing and merging.

**Comment by Jianing-Song** [comment]
@StevenClontz Add also that infinite earring is arc-connected and locally arc-connected, like S139?

**Pull request merged**